src/target/target_kind.cc

```diff
@@ -362,6 +362,8 @@ TVM_REGISTER_TARGET_KIND("rocm", kDLROCM)
     .set_target_parser(UpdateROCmAttrs);
 
 TVM_REGISTER_TARGET_KIND("opencl", kDLOpenCL)
+    .add_attr_option<Integer>("max_threads_per_block", Integer(256))
```

Comment:
I suppose that `max_num_threads` has the same meaning as `max_threads_per_block`, am I right?

Reply:
I think so, the main reason is that all other targets switched to the new flags